src/components/space-panel/components/space-rooms/room-label/room-label.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react"
+import {
+  StyledIcon,
+  Container,
+  RoomName,
+  IndicatorsContainer,
+  ErrorIndicator,
+  WarningIndicator,
+  UnreachableIndicator,
+} from "./styled"
+
+/* Counters are commented out, but might be used, UI is ready - if we have data */
+// const getAlarmsCount = (count: number) => (count > 99 ? "99+" : `${count}`)
+
+interface Props {
+  room: any
+}
+
+export const RoomLabel = ({ room }: Props) => {
+  const alarmCounter = room.alarmCounter || ({} as any)
+
+  const handleSelectRoom = () => {
+    // navigate to cloud room
+  }
+
+  return (
+    <Container onClick={handleSelectRoom}>
+      <StyledIcon name="room" />
+      <RoomName>{room.name}</RoomName>
```

Comment:
shouldn't they also have the same :hover effect as visited nodes?

Reply:
I don't see hovers at all in our designs. Ones that have it - it's because of rmwc defaults. I won't override anything unless there is a direct product issue with this behaviour.

Reply:
We should probably ask Javier about it. I commented on this issue because you have more experience with those components, and i'm not sure if we won't have to change how paddings work here (no padding on container, paddings on individual items)